journal/modernized/38-chapter-35.adoc

```diff
@@ -811,7 +812,7 @@ we were embarked on board a steamer; we proceeded down the spacious Potomac,
 as far as the head of Potomac Creek, which runs up from Chesapeake Bay.
 Here we were packed in crowded clumsy stages, and conveyed nine miles to Fredericsburgh,
 where we were again seated in a railway car, holding about sixty persons;
-many of them in rocking chairs, which seem quite a sine qua non with all classes here.
+many of them in rocking chairs, which seem quite a _sine qua non_ with all classes here.
```

Comment:
yup, this is good. I think all little greek/latin phrases we see should be in italics. 👍